addons/mail_client_extension/controllers/main.py

```diff
@@ -243,9 +273,50 @@ def res_partner_create(self, email, name, company, **kwargs):
             'name': name,
             'email': email,
         }
-        if company > -1:
+        if company and company > -1:
```

Comment:
What is this "-1" thing exactly? Seems strange.

Reply:
This is kept to ensure retro-compatibility. Because, rather then returning a null or undefined value for the company field, the older version of the plugin returns a `-1` to tell the controller that the company does not exist.

Reply:
Worth a small comment in that case, mentioning what it does and that we keep it for retro-compatibility.